src/timers.cc

```diff
@@ -12,50 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "timers.h"
```

Comment:
the related header should always be included first. https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes